src/UI/RenderConfig.elm

```diff
@@ -0,0 +1,49 @@
+module UI.RenderConfig exposing (RenderConfig, fromWindow, isMobile, isPortrait)
+
+import Element
+
+
+type alias RenderConfigData =
+    { deviceClass : Element.DeviceClass
+    , deviceOrientation : Element.Orientation
+    }
+
+
+type RenderConfig
+    = RenderConfig RenderConfigData
+
+
+fromWindow : { window | height : Int, width : Int } -> RenderConfig
+fromWindow window =
+    let
+        { class, orientation } =
+            Element.classifyDevice window
+    in
+    { deviceClass = class
+    , deviceOrientation = orientation
+    }
+        |> RenderConfig
+
+
+isMobile : RenderConfig -> Bool
+isMobile (RenderConfig { deviceClass }) =
+    -- TODO: Is Tablet mobile?
+    deviceClass == Element.Phone
+
+
+isPortrait : RenderConfig -> Bool
```

Comment:
I'm not sure if we should support landscape mobile screens 🤔

Reply:
I believe that applies for desktop users as well. There are vertical screens, and users who split their screen with two apps. In this last scenario, and as an example, a Navbar would be better than a Sidebar.

Reply:
Also, this serves as an example of how to correctly use this API, that is those `is` functions instead of casing through the internals...

Reply:
yeah, sounds good :D